flakehell/formatters/_baseline.py

```diff
@@ -8,8 +8,6 @@
 class BaseLineFormatter(BaseFormatter):
     def format(self, error):
         filename = error.filename
-        if filename.startswith('./'):
-            filename = filename[2:]
```

Comment:
It's better to have this thing because flake8 sometimes adds `./`, sometimes not, depends on what you've passed:

```
❯ flake8 example2.py
example2.py:1:1: F821 undefined name 'a'
❯ flake8 ./example2.py
./example2.py:1:1: F821 undefined name 'a'
```

What's the motivation to remove these lines?

Reply:
Ya know, I don't quite remember. Let me play around with the tool some to see if anything jogs my memory.

Reply:
Ah, I remember now. Since this immediately calls into `make_baseline` which already does the stripping, this code is unnecessary.